pkg/network/ovn_kubernetes_test.go

```diff
@@ -139,6 +141,141 @@ func TestRenderOVNKubernetesIPv6(t *testing.T) {
 	g.Expect(script).To(ContainSubstring("pssl:9641:[::]"))
 }
 
+func TestRenderedOVNKubernetesConfig(t *testing.T) {
+	type testcase struct {
+		desc                string
+		expected            string
+		hybridOverlayConfig *operv1.HybridOverlayConfig
+	}
+	testcases := []testcase{
+		{
+			desc: "default",
+			expected: `[default]
+mtu="1500"
```

Comment:
so as a hack, you could add a newline immediately after the backtick, to get `[default]` lined up with everything else, and likewise in the other cases, and then just update the comparison to use `strings.TrimSpace(tc.expected)` to fix it up...

Reply:
done